Sources/Logging/Logging.swift

```diff
@@ -156,32 +335,78 @@ public enum LoggingSystem {
 }
 
 extension Logger {
+    /// `Metadata` is a typealias for `[String: Logger.Metadata.Value]` the type of the metadata storage.
     public typealias Metadata = [String: MetadataValue]
 
+    /// A logging metadata value. `Logger.Metadata.Value` is string, array, and dictionary literal convertible.
     public enum MetadataValue {
+        /// A metadata value which is a `String`.
         case string(String)
+
+        /// A metadata value which is some `CustomStringConvertible`.
         case stringConvertible(CustomStringConvertible)
+
+        /// A metadata value which is a dictionary from `String` to `Logger.Metadata.Value`.
         case dictionary(Metadata)
+
+        /// A metadata value which is an array of `Logger.Metadata.Value`s.
         case array([Metadata.Value])
     }
 
+    /// The log level.
     public enum Level: Int {
+        /// The `.debug` log level, appropriate for messages that contain information normally of use only when
```

Comment:
Not sure the `The `.debug` log level,` are helpful.

Maybe just the "Appropriate for..."

?